src/components/verification-grid-tile/verification-grid-tile.ts

```diff
@@ -303,7 +303,14 @@ export class VerificationGridTileComponent extends SignalWatcher(AbstractCompone
       >
         ${this.keyboardShortcutTemplate()}
         <figure class="spectrogram-container ${figureClasses}">
-          <figcaption class="tag-label">${this.model?.tag?.text ?? this.model?.tag}</figcaption>
+          <div class="figure-head">
+            <figcaption class="tag-label">${this.model?.tag?.text ?? this.model?.tag}</figcaption>
```

Comment:
this might be a good time to add a tooltip in: "This item was tagged as 'xxx' in your data source"

Reply:
Completed version

![image](https://github.com/user-attachments/assets/a8f7f710-4e58-451b-9de2-53305733f2fa)